nautobot/core/urls.py

```diff
@@ -62,7 +62,7 @@
 
 if settings.METRICS_ENABLED:
     urlpatterns += [
-        path("", include("django_prometheus.urls")),
+        path("metrics/", custom_app_metric_view),
```

Comment:
Does this cause us to *lose* the current django_prometheus metrics, or is it only augmenting those?

Reply:
That is correct. If you compare the sources between the `django_prometheus`-provided URL and `custom_app_metric_view` then there are only two lines added which register an _additional_ collector into the view (which is our app extensibility collector).